engine/src/main/java/com/datatorrent/stram/client/StramClientUtils.java

```diff
@@ -119,8 +119,8 @@
   @Deprecated
   public static final String KEY_TAB_FILE = StramUserLogin.DT_AUTH_PREFIX + "store.keytab";
   public static final String TOKEN_ANTICIPATORY_REFRESH_FACTOR = StramUserLogin.DT_AUTH_PREFIX + "token.refresh.factor";
-  public static final long DELEGATION_TOKEN_MAX_LIFETIME_DEFAULT = 7 * 24 * 60 * 60 * 1000;
-  public static final long DELEGATION_TOKEN_RENEW_INTERVAL_DEFAULT = 24 * 60 * 60 * 1000;
+  public static final long RM_DELEGATION_TOKEN_MAX_LIFETIME_DEFAULT = 7 * 24 * 60 * 60 * 1000;
```

Comment:
Suggest you submit this change in a separate PR as it is not related to the Hadoop 3.1 change.

Reply:
I seem to remember Intellij complaining about those two lines after I made the changes to the constants used by Hadoop 3.1.  Perhaps I misunderstood/misinterpreted the relationship?